inngest/src/main/kotlin/com/inngest/InngestFunction.kt

```diff
@@ -20,13 +20,7 @@ abstract class InngestFunction {
     }
 
     fun id(): String {
-        try {
-            return buildConfig().id!!
-        } catch (e: Exception) {
-            throw InngestInvalidConfigurationException(
-                "Function id must be configured via builder: ${this.javaClass.name}",
-            )
-        }
```

Comment:
this is unnecessary because it's being thrown here https://github.com/inngest/inngest-kt/blob/70ac4e372eba98d579ce60969cd0db2933676062/inngest/src/main/kotlin/com/inngest/InngestFunctionConfigBuilder.kt#L210-L212

and `catch (e: Exception)` was swallowing other `InngestInvalidConfigurationException`s and rethrowing them for the wrong reason